Sources/docc/DocCDocumentation.docc/linking-to-symbols-and-other-content.md

```diff
@@ -0,0 +1,198 @@
+# Linking to Symbols and Other Content
+
+Facilitate navigation between pages using links.
+
+## Overview
+
+DocC supports the following link types to enable navigation between pages:
+
+| Type | Usage |
+| --- | --- |
+| Symbol | Links to a symbol's reference page in your documentation. |
+| Article | Links to an article or API collection in your documentation catalog. |
+| Tutorial | Links to a tutorial in your documentation catalog. |
+| Web | Links to an external URL. |
+
+### Navigate to a Symbol
+
+To add a link to a symbol, wrap the symbol's name in a set of double backticks 
+(\`\`).
+
+```markdown
+``SlothCreator``
+```
+
+For nested symbols, include the path to the symbol in the link.
+
+```markdown
+``SlothCreator/Sloth/eat(_:quantity:)``
+```
+
+DocC resolves symbol links relative to the context they appear in. For example, 
+a symbol link that appears inline in the `Sloth` class, and targets a 
+symbol in that class, can omit the `SlothCreator/Sloth/` portion of the symbol 
+path.
+
+In some cases, a symbol's path isn't unique, such as with overloaded methods in 
```

Comment:
Recommend adding a description here of what you mean by "overloaded".

Reply:
I find it unnecessary since 'overload' is a common term in this field.